src/azure-cli/azure/cli/command_modules/keyvault/_help.py

```diff
@@ -238,7 +238,13 @@
 helps['keyvault create'] = """
 type: command
 short-summary: Create a Vault or HSM.
-long-summary: If `--enable-rbac-authorization` is not specified, then default permissions are created for the current user or service principal unless the `--no-self-perms` flag is specified.
+long-summary: >-
+    RBAC authorization is enabled by default. If `--enable-rbac-authorization` is manually specified to `false` and
+    `--no-self-perms` flag is not specified, default permissions are created for the current user or service principal.
+
+
```

Comment:
Could we just leave a blank line in the middle?  Is there any reason for leaving two blank lines?

Reply:
Got it, thanks!